client/lib/cart-values/cart-items.js

```diff
@@ -406,6 +406,16 @@ export function hasDomainRegistration( cart ) {
 	return some( getAllCartItems( cart ), isDomainRegistration );
 }
 
+/**
+ * Determines whether there is at least one domain registration item in the specified shopping cart.
+ *
+ * @param {Object} cart - cart as `CartValue` object
+ * @returns {boolean} true if there is at least one domain registration item, false otherwise
+ */
+export function hasOnlyDomainProducts( cart ) {
+	return every( getAllCartItems( cart ), isDomainProduct );
```

Comment:
Well, that's intended. 

If there are only domains in the cart (hence the `every`), we skip any troubles and just don't show the test. So in practice, if you only have domains in your cart you're out of the test. I found it easier for the implementation, since if its `some` instead we'd have a third variation: No seal (because there's no 30 day MBG for domains) but yes text "96 hours guarantee for domains" and I didn't want to include this variation so I thought to better leave domains alone out of this.

Thoughts?